laalaa/apps/advisers/healthchecks.py

```diff
@@ -0,0 +1,43 @@
+from django.conf import settings
+from moj_irat.healthchecks import HealthcheckResponse, UrlHealthcheck, registry
+
+class CeleryWorkersHealthcheck(object):
+    def __init__(self, name):
+        self.name = name
+    
+    def __call__(self, *args, **kwargs):
+        from celery import Celery
+        
+        try:
+            app = Celery('laalaa')
+            stats = app.control.inspect().stats()
+            if not stats:
+                return self.error_response('No running workers were found.')
+            workers = stats.values()
+            if len(workers) == 0:
```

Comment:
🙋‍♂️ It seems `len(workers) == 0` and `not workers` are equivalent in Python:

```
>>> list = []
>>> len(list) == 0
True
>>> not list
True

>>> list = [1]
>>> len(list) == 0
False
>>> not list
False
```

Reply:
Nice!